tests/unit/states/archive_test.py

```diff
@@ -72,15 +67,9 @@ def test_extracted_tar(self):
                                                    'file.file_exists': mock_false,
                                                    'file.makedirs': mock_true,
                                                    'cmd.run_all': mock_run}):
-                    if HAS_PWD:
-                        running_as = pwd.getpwuid(os.getuid()).pw_name
```

Comment:
@s0undt3ch I see that you added in the ``HAS_PWD`` stuff in 1181a50, so perhaps there's something I'm missing. I can tell you, though, that the tests failed when run under the mockbuild user, due to the filename not containing "_mockbuild".